brightray/browser/browser_main_parts.cc

```diff
@@ -256,11 +253,29 @@ void BrowserMainParts::PreMainMessageLoopStart() {
 }
 
 void BrowserMainParts::PreMainMessageLoopRun() {
+  auto command_line = base::CommandLine::ForCurrentProcess();
+
+  const auto enable_features =
+      command_line->GetSwitchValueASCII(switches::kEnableFeatures);
+  const auto disable_features =
+      command_line->GetSwitchValueASCII(switches::kDisableFeatures);
+
+  std::unique_ptr<base::FeatureList> feature_list(new base::FeatureList);
```

Comment:
use `auto feature_list = std::make_unique<base::FeatureList>();` instead

Reply:
We can't use that in this branch yet:

```
../../brightray/browser/browser_main_parts.cc:263:28: error: no member named 'make_unique' in namespace 'std'
  auto feature_list = std::make_unique<base::FeatureList>();
```